Add function to insert viewport dimensions into WMDESizeIssueEvents

Abban opened the pull request:
This allows event maps to insert the viewport dimensions
into WMDESizeIssueEvents. It is unused in this commit, and
will be need to be rebased and used in each banner's event
map branch once merged.

Ticket: https://phabricator.wikimedia.org/T349187

timEulitz commented on campaign_info.toml:
```diff
@@ -88,30 +88,11 @@ campaign_tracking = "wpde-04-181214"
+[wikipediade.banners.ctrl]
+filename = "./wikipedia.de_prototype/banner_ctrl.js"
 pagename = "B18WPDE_04-181214_fixed_large_ctrl"
 tracking = "wpde-04-181214-fixed-large-ctrl"
```
These still need to be updated to `B19WPDE_01-191111_ctrl`

Abban committed:
Remove unused method from size issue tracker

These are an artifact of the old code base and not
being used anymore.

Ticket: https://phabricator.wikimedia.org/T349187

Abban committed:
Add createViewportInfo function

This allows event maps to insert the viewport dimensions
into WMDESizeIssueEvents. It is unused in this commit, and
will be need to be rebased and used in each banner's event
map branch once merged.

Ticket: https://phabricator.wikimedia.org/T349187
